pipeline/scripts/4/3/tier-3/tests_rgw_tier-2_bucket-notifications.sh

```diff
@@ -0,0 +1,50 @@
+#! /bin/sh
```

Comment:
Could we rename the script name? Here, the filename is used to describe the suite that is being executed. It is the one that we use in the emails.

Hence, it is not required to add `tier-2`. Also, we suggest not use `_` instead opt to use `-` underscores

Reply:
Example: `test-rgw-bucket-notifications.sh`

Reply:
Noted, renamed the file.